slack/web/client.py

```diff
@@ -2154,3 +2172,150 @@ def views_publish(
+            team_id (str): The workspace to create the channel in.
+                Note: this argument is required unless you set org_wide=true.
+
+        """
+        if not org_wide and team_id is None:
+            raise e.SlackRequestError("team_id is required if org_wide is False")
+        kwargs.update(
+            {
+                "is_private": is_private,
+                "name": name,
+                "org_wide": org_wide,
+                "team_id": team_id,
+            }
+        )
+        return self.api_call("admin.conversations.create", json=kwargs)
+
+    def admin_conversations_delete(
+        self, channel_id: str, **kwargs
+    ) -> Union[Future, SlackResponse]:
+        """Delete a public or private channel.
+
+        Args:
+            channel_id (str): The channel to delete.
+
+        """
+        kwargs.update({"channel_id": channel_id})
+        return self.api_call("admin.conversations.delete", json=kwargs)
+
+    def admin_conversations_invite(
+        self, channel_id: str, user_ids: Union[str, List[str]], **kwargs
+    ) -> Union[Future, SlackResponse]:
+        """Invite a user to a public or private channel.
+
+        Args:
+            channel_id (str): The channel that the users will be invited to.
+            user_ids (str or list): The users to invite.
+        """
+        kwargs.update({"channel_id": channel_id})
+        if isinstance(user_ids, list):
+            kwargs.update({"channel_ids": ",".join(user_ids)})
+        else:
+            kwargs.update({"channel_ids": user_ids})
+        return self.api_call("admin.conversations.invite", json=kwargs)
+
+    def admin_conversations_archive(
+        self, channel_id: str, **kwargs
+    ) -> Union[Future, SlackResponse]:
+        """Archive a public or private channel.
+
+        Args:
+            channel_id (str): The channel to archive.
+        """
+        kwargs.update({"channel_id": channel_id})
+        return self.api_call("admin.conversations.archive", json=kwargs)
+
+    def admin_conversations_unarchive(
+        self, channel_id: str, **kwargs
+    ) -> Union[Future, SlackResponse]:
+        """Unarchive a public or private channel.
+
+        Args:
+            channel_id (str): The channel to unarchive.
+        """
+        kwargs.update({"channel_id": channel_id})
+        return self.api_call("admin.conversations.unarchive", json=kwargs)
+
+    def admin_conversations_rename(
+        self, channel_id: str, name: str, **kwargs
+    ) -> Union[Future, SlackResponse]:
+        """Rename a public or private channel.
+
+        Args:
+            channel_id (str): The channel to rename.
+            name (str): The name to rename the channel to.
+        """
+        kwargs.update({"channel_id": channel_id, "name": name})
+        return self.api_call("admin.conversations.rename", json=kwargs)
+
+    def admin_conversations_search(self, **kwargs) -> Union[Future, SlackResponse]:
```

Comment:
I thought it would be cool to add something like 
```python
if "team_ids" in kwargs:
    if isinstance(kwargs.get("team_ids"), list):
        kwargs.update({"team_ids": ",".join(team_ids)})
```
to make it easier to pass in the team ids as kwargs, but wasn't sure if it was wanted since it's an optional argument

Reply:
I think we should not change the current behavior and should not have the conversion here (as it's not a required parameter). However, we may want to consider having a global converter for list objects in `json` parameter. As the Slack API always accepts comma-separated values as a single string, that's a reasonable option.